Cohere/SampleRequestsAndResponses/SampleChatRequests.cs

```diff
@@ -0,0 +1,163 @@
+using Cohere.Types;
+
+namespace Cohere.SampleRequestsAndResponses;
+
+/// <summary>
+/// Examples of valid and invalid chat requests to the Cohere API
+/// </summary>
+public class SampleChatRequests
```

Comment:
is any of these SampleRequests used in the current Cohere project? If not, it looks like they need to be moved to the test project